packages/ui/cart/drawer/Upsell.tsx

```diff
@@ -42,14 +44,19 @@ export function Upsell({ item }: UpsellProps) {
             borderColor="brand.200"
             borderStyle="solid"
          >
-            <Link href={item.url} color="brand.500" fontWeight="semibold">
-               {item.marketingTitle ?? item.name}
-            </Link>
-            <Text color="brand.500">Add the product below to your order</Text>
+            <NextLink href={`/products/${item.handle}`} passHref>
+               <LinkOverlay
+                  id={item.handle}
+                  fontWeight="semibold"
+                  color="brand.500"
+                  _groupHover={{ color: 'black' }}
```

Comment:
What does this _groupHover achieve? I'm not that familiar with it.

If this is to style the underline on hover, I'd say we should just set `textDecoration="none"` because I don't think we show underline on hover in the rest of our product cards.

Reply:
it changes the color of the text, which allows for a visual indication that there is a link to click basically.